doc/robots.txt

```diff
@@ -1,2 +1,3 @@
 User-agent: *
 Disallow:
+Sitemap: https://docs.librenms.org/sitemap.xml
```

Comment:
How come you've added this in?

We disallow bots so they won't see this value anyway.

Reply:
I know almost 0 about SEO but:

`User-agent: *` means all robots are allowed
`Disallow:` Nothing is disallowed to index

So I've added the sitemap in a hope of Google indexing the docs site so when someone search about librenms and is covered in the docs it will return docs as result